### DISCUSSION TITLE
Inquiry Regarding Suspended GitHub Account (Username: amankr1469)

### DISCUSSION THREAD

**Discussion by Coderx85** [community/community#108056] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello GitHub Team,

I hope this message finds you well. I am reaching out on behalf of my friend whose GitHub account with the username "amankr1469" was recently suspended without any prior notice or explanation. This suspension has caused a significant disruption to my friend's ongoing projects and collaborations.

We have thoroughly reviewed GitHub's Terms of Service and community guidelines and are unable to identify any potential violations from my friend's end. As a result, we are seeking clarification on the reason behind the suspension and guidance on the resolution process.

We understand the importance of maintaining a respectful and compliant community on GitHub, and we are committed to adhering to all policies and terms. Therefore, we kindly request your assistance in reviewing this matter and providing the necessary steps to address the suspension.

Your prompt attention to this issue would be greatly appreciated, as the suspended account is integral to my friend's professional and academic commitments.

Best regards,
Priyanshu
[Priyanshu085](https://www.github.com/Priyanshu085)

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @Priyanshu085 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

Appreciate your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
